package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "bindings": "1.2.1",
-    "nan": "2.0.7"
+    "nan": "^2.0.7"
```

Comment:
Emergency or not I agree with @defunctzombie. For future compatibility this definitely should specify a static version before it can be released. I'll create a PR to fix this.